src/adapters/TelnetAdapter.ts

```diff
@@ -727,20 +763,9 @@ export class TelnetAdapter {
                         break;
                     }
                 }
-                //if the next-to-last line of collection is `...`, then scrap the values
-                //because we will need to run a full evaluation (later) to get around the `...` issue
-                if (collectionLineList.length > 3 && collectionLineList[collectionLineList.length - 2].trim() === '...') {
```

Comment:
These 3 if blocks used to parse the array or object children directly from the printed output, thus avoiding another telnet call. However, in this situation a roInt would still be printed as "roInt" (which is what we are fixing in this PR). This means, unfortunately, that this logic needs to be scrapped, and every inner object or array needs to be printed in its own for loop. I don't expect much performance impacts because we could only take the shortcut on small objects/arrays anyway.